java/src/main/kotlin/io/spine/protodata/java/ClassName.kt

```diff
@@ -101,6 +101,21 @@ public class ClassName(
     public fun nested(simpleClassName: String): ClassName =
         ClassName(packageName, simpleNames + simpleClassName)
 
+    /**
+     * Obtains a new `ClassName` with the last element removed from the list of simple names.
+     *
+     * The method is useful for obtaining names for outer classes,
+     * e.g. `Message` from `Message.Builder`.
+     *
+     * @throws IllegalStateException when the current class name is the top level class name.
```

Comment:
Please capitalise `when`. Have a look at the [documentation conventions](https://github.com/SpineEventEngine/documentation/wiki/KDoc-and-Dokka#documenting-a-type) we use.